NOTICES.txt

```diff
@@ -1,374 +1,141 @@
+The following is a listing of the conjur open source components detailed
+in this document. This list is provided for your convenience; please read
+further if you wish to review the copyright notice(s) and the full text
+of the license associated with each component.
 
-http://www.apache.org/licenses/
+SECTION 1: Apache License 2.0
 
-TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+>>> https://rubygems.org/gems/gli/versions/2.17.1
+>>> https://rubygems.org/gems/aws-sdk-iam/versions/1.4.0
+>>> https://rubygems.org/gems/conjur-api/versions/5.1.0
 
-1. Definitions.
+SECTION 2: BSD 2-clause "Simplified" License
+
+>>> https://rubygems.org/gems/pg/versions/0.20.0
+
+SECTION 3: BSD 3-clause "New" or "Revised" License
+
+>>> https://rubygems.org/gems/puma/versions/3.12.2
+>>> https://rubygems.org/gems/ffi/versions/1.9.25
+
+SECTION 4: MIT License
+
+>>> https://rubygems.org/gems/command_class/versions/0.0.2
+>>> https://rubygems.org/gems/base58/versions/0.2.3
+>>> https://rubygems.org/gems/iso8601/versions/0.10.1
+>>> https://rubygems.org/gems/nokogiri/versions/1.10.5
+>>> https://rubygems.org/gems/rack/versions/1.6.12
+>>> https://rubygems.org/gems/rails/versions/4.2.11
+>>> https://rubygems.org/gems/rails-api/versions/0.4.1
+>>> https://rubygems.org/gems/rake/versions/13.0.1
+>>> https://rubygems.org/gems/sprockets/versions/3.7.2
+>>> https://rubygems.org/gems/sequel-postgres-schemata/versions/0.1.1
+>>> https://rubygems.org/gems/sequel-rails/versions/0.9.15
+>>> https://rubygems.org/gems/activesupport/versions/4.2.11
+>>> https://rubygems.org/gems/listen/versions/3.0.6
+>>> https://rubygems.org/gems/slosilo/versions/2.1.1
+>>> https://rubygems.org/gems/loofah/versions/2.3.1
+>>> https://rubygems.org/gems/ruby_dep/versions/1.3.1
+>>> https://rubygems.org/gems/simplecov/versions/0.14.1
+>>> https://rubygems.org/gems/sass-rails/versions/5.0.6
+>>> https://rubygems.org/gems/therubyracer/versions/0.12.3
+>>> https://rubygems.org/gems/uglifier/versions/3.2.0
+>>> https://rubygems.org/gems/autoprefixer-rails/versions/7.1.2.3
+>>> https://rubygems.org/gems/bootstrap-sass/versions/3.4.1
+>>> https://rubygems.org/gems/dry-struct/versions/0.4.0
+>>> https://rubygems.org/gems/font-awesome-sass/versions/4.7.0
+>>> https://rubygems.org/gems/net-ldap/versions/0.16.1
+>>> https://rubygems.org/gems/rails_12factor/versions/0.0.3
+>>> https://rubygems.org/gems/kubeclient/versions/3.1.2
+>>> https://rubygems.org/gems/websocket-client-simple/versions/0.3.0
+>>> https://rubygems.org/gems/openid_connect/versions/1.1.6
+>>> https://rubygems.org/gems/conjur-policy-parser
+>>> https://rubygems.org/gems/conjur-rack/versions/3.1.0
+>>> https://rubygems.org/gems/conjur-rack-heartbeat/versions/2.0.0
+>>> https://rubygems.org/gems/rack-rewrite/versions/1.5.1
+>>> https://rubygems.org/gems/coffee-rails
+>>> https://rubygems.org/gems/bcrypt-ruby/versions/3.0.1
+
+SECTION 5: Gems with no/unknown License
+
+>>> https://rubygems.org/gems/base32-crockford/versions/0.1.0
```

Comment:
Opening this Gem up, it looks like it's licensed under the Ruby license:
```
# (c) 2008, Levin Alexander <http://levinalex.net>
#
# This file is released under the same license as ruby.
```
It looks like we can put it under "2 Clause BSDL": https://www.ruby-lang.org/en/about/license.txt

Reply:
Gems can be opened after installing locally with:
```
$ gem install base32-crockford --no-document
$ gem unpack base32-crockford
Unpacked gem: '/Users/jasonvanderhoof/code/openshift-dap-operator/base32-crockford-0.1.0'
$ atom /Users/jasonvanderhoof/code/openshift-dap-operator/base32-crockford-0.1.0
```